internal/db/schema/migrations/oss/postgres_56_02_test.go

```diff
@@ -65,12 +64,9 @@ func TestMigrations_NewForeignKeys(t *testing.T) {
 
 	// Create project
 	wrapper := db.TestWrapper(t)
-	iamRepo := iam.TestRepo(t, conn, wrapper)
 	kmsCache := kms.TestKms(t, conn, wrapper)
-
-	// Create a session
-	sess := session.TestSession(t, conn, wrapper, session.TestSessionParams(t, conn, wrapper, iamRepo))
-	_ = sess
```

Comment:
This became a pain now since we have to do it manually, so I removed it. It doesn't add that much confidence that I think it's worth trying to figure out doing it manually, but I can spend the time if we insist.